Eask

```diff
@@ -0,0 +1,28 @@
+(package "elsa"
+         "0.1.0"
+         "Emacs Lisp Static Analyser")
+
+(website-url "https://github.com/emacs-elsa/Elsa")
+(keywords "languages" "lisp")
+
+(package-file "elsa.el")
+
+(files "elsa-*.el" "bin/elsa*")
+
+(script "test" "echo \"Error: no test specified\" && exit 1")
+
+(source 'gnu)
+(source 'melpa)
+
+(depends-on "emacs" "26.1")
+(depends-on "dash" "2.14")
+(depends-on "seq" "0")
+(depends-on "f" "0")
+(depends-on "trinary" "1.0.0")
+
+(development
+ (depends-on "elsa")
+ (depends-on "undercover")
+ (depends-on "buttercup"))
+
+(setq network-security-level 'low)  ; see https://github.com/jcs090218/setup-emacs-windows/issues/156#issuecomment-932956432
```

Comment:
Isn't this dangerous?  I skimmed the issue and it seems to be a windows problem (?).  If so, is it possible to guard this by the os type?

Reply:
Yeah, it's often bad since we are skipping most network security checks. I chose this since I normally operate tasks in GHA, and not on my own PC.

Another solution for this, is to update DST Root CA X3 cert manually:

```
    # Remove expired DST Root CA X3 certificate. Workaround for
    # https://debbugs.gnu.org/cgi/bugreport.cgi?bug=51038 bug on Emacs 27.2.
    # https://github.com/jcs090218/setup-emacs-windows/issues/156#issuecomment-1126671598
    - name: Workaround for Emacs 27.2's Windows build from GNU FTP
      if: ${{ runner.os == 'Windows' && matrix.emacs-version == '27.2' }}
      run: |
        gci cert:\LocalMachine\Root\DAC9024F54D8F6DF94935FB1732638CA6AD77C13
        gci cert:\LocalMachine\Root\DAC9024F54D8F6DF94935FB1732638CA6AD77C13 | Remove-Item
```

Reply:
Is the Eask file regular elisp?  We could then wrap it with `(when (eq system-type 'windows-nt) ...)`

Reply:
Yeah, that would work too! Do you prefer doing that?

I thought your question is to not use `(setq network-security-level 'low)` entirely. If we do `(when (eq system-type 'windows-nt) ...)`, then it will only effect Windows users for sure.

Reply:
Nah, I think fixing it in the GHA build definition is cleaner.